ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs

```diff
@@ -11,39 +11,72 @@
 
 namespace ASPNetCoreMastersTodoList.Api.Controllers
 {
+    [ApiController]
+    [Route("items")]
     public class ItemsController : Controller
     {
         private readonly ILogger<ItemsController> _logger;
         private ItemService _itemService;
 
-        public ItemsController(ILogger<ItemsController> logger, ItemService itemService)
+        public ItemsController(ILogger<ItemsController> logger)
         {
+            _itemService = new ItemService();
             _logger = logger;
-            _itemService = itemService;
         }
 
         [HttpGet]
-        IEnumerable<string> GetAll(int userId)
+        IActionResult Get()
         {
-            return _itemService.GetAll(userId);
+            var result = _itemService.GetAll();
+            return Ok(result);
         }
 
         [HttpGet]
-        int Get(int userId)
+        [Route("{itemId}")]
+        IActionResult Get(int id)
```

Comment:
parameter should match what's in the route template

Reply:
thank you for reviewing, this was updated.